xml/System/Index.xml

```diff
@@ -167,9 +190,9 @@
         <Parameter Name="value" Type="System.Int32" />
       </Parameters>
       <Docs>
-        <param name="value">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="value">The index value from the end.</param>
```

Comment:
```suggestion
        <param name="value">The index value from the end. of a collection</param>
```

Reply:
I modified your suggestion to move the dot to the end.